inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/flink/doris/load/DorisRespond.java

```diff
@@ -0,0 +1,42 @@
+package org.apache.inlong.sort.flink.doris.load;
+
+/**
+ * @program: incubator-inlong
```

Comment:
After the class function is annotated, the author and creation time need to be removed